net-libs/libtorrent/libtorrent-0.14.0.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools toolchain-funcs
+
+DESCRIPTION="BitTorrent library written in C++ for *nix"
+HOMEPAGE="https://rakshasa.github.io/rtorrent/"
+SRC_URI="https://github.com/rakshasa/rtorrent-archive/raw/master/${P}.tar.gz"
```

Comment:
whoa this is weird. It looks like the dev now has that repo with archives in it, weird. The hash of that file does match exactly with `https://github.com/rakshasa/rtorrent/releases/download/v0.10.0/libtorrent-0.14.0.tar.gz` 
the problem with that is the version of the release is the rtorrent version not the libtorrent one :(. I guess keeping the rtorrent-archive link is fine, im going to add a comment tho i think in case future people dont think its weird too

Reply:
Oh, I didn't pay attention. tags from github instead ?
https://github.com/rakshasa/libtorrent/archive/refs/tags/v0.14.0.zip

"https://github.com/rakshasa/${PN}/archive/refs/tags/v${PV}.zip"

I didn't check diffs for now.